### PR TITLE
Feat multi fetch

### PR DESCRIPTION
**Pull request by petrjasek**
implementing multi fetch + fetchAs

### CODE REVIEW

File: client/app/scripts/superdesk-archive/module.js [hunk 1]
```diff
@@ -57,9 +57,15 @@ define([
             this.count = 0;
         };
 
+        this.items = getItems;
```
Comment (plamut) [inline client/app/scripts/superdesk-archive/module.js]: What's with this public alias for the `getItems` function? Or was it perhaps meant to be a function call, e.g. `getItems()`?

**Comment by plamut** [comment]
Looking good so far IMO.

File: client/app/scripts/superdesk-archive/module.js (hunk 1 above)
Comment (petrjasek) [inline client/app/scripts/superdesk-archive/module.js]: yep exactly. but it's kind of weird when mixed together with `this.reset = function () { ... };`
will clean this a bit

File: client/app/scripts/superdesk-archive/module.js [hunk 2]
```diff
@@ -57,6 +57,13 @@ define([
             this.count = 0;
         };
 
+        /**
+         * Get list of selected items
+         */
+        this.getItems = function getItems() {
```
Comment (plamut) [inline client/app/scripts/superdesk-archive/module.js]: Method name redundant here?

Comment (petrjasek) [inline client/app/scripts/superdesk-archive/module.js]: it's better for stack traces / profiling

Comment (plamut) [inline client/app/scripts/superdesk-archive/module.js]: Yeah, but don't stack traces already contain the name that is assigned to the method when it is bound to controller?

Comment (petrjasek) [inline client/app/scripts/superdesk-archive/module.js]: yep you're right, no need for the names there

**Comment by ioanpocol** [comment]
:+1:

**Pull request merged**